src/sentry/models/actor.py

```diff
@@ -1,90 +0,0 @@
-from __future__ import annotations
-
-from django.conf import settings
-from django.db import models, router, transaction
-from django.forms import model_to_dict
-
-from sentry.backup.dependencies import ImportKind, PrimaryKeyMap
-from sentry.backup.helpers import ImportFlags
-from sentry.backup.scopes import ImportScope, RelocationScope
-from sentry.db.models import Model, region_silo_model
-from sentry.db.models.fields.foreignkey import FlexibleForeignKey
-from sentry.db.models.fields.hybrid_cloud_foreign_key import HybridCloudForeignKey
-from sentry.models.outbox import OutboxCategory, OutboxScope, RegionOutbox, outbox_context
-
-ACTOR_TYPES = {
-    "team": 0,
-    "user": 1,
-}
-
-
-@region_silo_model
-class Actor(Model):
-    # Temporary until Actor is removed
-    __relocation_scope__ = RelocationScope.Excluded
-
-    type = models.PositiveSmallIntegerField(
-        choices=(
-            (ACTOR_TYPES["team"], "team"),
-            (ACTOR_TYPES["user"], "user"),
-        )
-    )
-    user_id = HybridCloudForeignKey(
-        settings.AUTH_USER_MODEL, on_delete="CASCADE", db_index=True, unique=True, null=True
-    )
-    team = FlexibleForeignKey(
-        "sentry.Team",
-        related_name="actor_from_team",
-        db_constraint=True,
-        db_index=True,
-        unique=True,
-        null=True,
-        on_delete=models.CASCADE,
-    )
```

Comment:
I think we need a separate pr first that drops the constraint, otherwise we can hit errors when attempting to delete teams

Reply:
Or actually, it'd be fine to drop the constraint in this migration as well

Reply:
I removed the foreign key between Team -> Actor foreign key in #70090, I'll remove this one now so that both sides are gone.